bigbluebutton-html5/imports/api/audio/client/bridge/kurento.js

```diff
@@ -11,7 +11,14 @@ const GLOBAL_AUDIO_PREFIX = 'GLOBAL_AUDIO_';
 const logFunc = (type, message, options) => {
   const topic = options.topic || 'audio';
 
-  logger[type](`[${topic}] ${JSON.stringify(message)} options:${JSON.stringify(options)}`);
+  logger[type]({
+    logCode: 'TODO_kurento_audio',
+    extraInfo: {
+      topic,
+      message,
+      options,
+    },
+  }, options && options.message && options.message.id ? options.message.id : 'kurento audio');
```

Comment:
Shouldn't the line be `), message);`? Is the message contained in the options object?

Reply:
I was looking for a string to display. A group of messages have a decently descriptive `options.message.id` but not all. For the rest - I add `'kurento audio'` so we have something

Reply:
Don't they all pass in the `message` parameter though? Seems easiest to just use that

Reply:
`"message":"Sending message:",`
vs
`options.message.id: "start",` with topic: 'audio'
Neither is great but I find the options.message.id a bit more meaningful

Reply:
I didn't realize that both options were pretty bad.